shell/mkcd.sh

```diff
@@ -0,0 +1,9 @@
+function_test(){
+    name="$(whoami)"
+    mkdir "$name-folder"
+    cd "$name-folder"
+    touch .gitkeep
+}
+
+#!/bin/bash
```

Comment:
Shebang needs to be at the beginning of the file 
[Cf Stackoverflow](https://stackoverflow.com/a/12911604/11018979)